PlayerInfo/Dolby/AMLogic/Implementation.cpp

```diff
@@ -0,0 +1,144 @@
+/*
+ * If not stated otherwise in this file or this component's LICENSE file the
+ * following copyright and licenses apply:
+ *
+ * Copyright 2020 RDK Management
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "../../Module.h"
+#include "Dolby.h"
+#include "audio_if.h"
+#include <string.h>
+#include "hardware/audio.h"
```

Comment:
@anand-ky : We're introducing amlogic-specific libraries and calls here. Is this okay? If I think back to Aug 2020, there was discussion about the need to maintain rdkservices as free from SoC-specific libraries inclusions and calls..